### ISSUE TITLE
Share via Browser: allow multiple file / folder upload

### ISSUE THREAD

**Issue by apfelchips** [trebleshot/android#137]
**Is your feature request related to a problem? Please describe.**
Currently you can't upload multiple files / folders on the HTTP share provided by TrebleShot. Since TrebleShot is currently only available on android sharing files to other platforms is difficult / cumbersome.

**Describe the solution you'd like**
Allow uploading of multiple files from the HTML-input form and handle multiple files.

**Describe alternatives you've considered**
Implement the new HTML5 drag&drop / folder upload api and have input-form as a backup for legacy clients.

**Additional context**
example folder upload implementation: 
<https://stackoverflow.com/a/53058574>
NanoHTTPD supports multi file/part upload: <https://github.com/NanoHttpd/nanohttpd/blob/master/fileupload/src/test/java/org/nanohttpd/junit/fileupload/TestNanoFileUpLoad.java#L182>

**Comment by velitasali** [comment]
This is something I was thinking of, but this needs improvement on the web share interface, which is a low-priority task on my to-do list.

**Comment by mubashir-rehman** [comment]
this might have been fixed with latest commit. i think.

**Comment by velitasali** [comment]
Nope. This needs some js to work. I have some ideas on how I should do it, but for now, it needs to wait.

**Comment by jerryn70** [comment]
@velitasali curently we can receive files from other device that dont have trebleshot installed via the share via browser option. But how can i sent files to other device that dont have trebleshot installed ?
Anything im missing or is it not possible ?

**Comment by mubashir-rehman** [comment]
@jerryn70 you can share the files using the same feature from trebleshot. just select the files and 
1)click in 🌐 on version 2.0
2)cick send button and then select share on browser o n version 1.4.4

**Comment by jerryn70** [comment]
@mubashir-rehman yes got it thank you. BTW why its showing wi-fi and vpn server ? 2 options with different url . 

Wifi is for local hotspot connection and vpn server is to send device that are connected to the same network ? is it?
